geopy/geocoders/googlev3.py

```diff
@@ -179,6 +180,11 @@ def geocode(
             .. versionchanged:: 1.14.0
                Now ``query`` is optional if ``components`` param is set.
 
+        :param str place_id: Address details are obtained using google
+                             place id.It is not used with the query or bounds
```

Comment:
1. There should be a space: `s/place id.It is/place id. It is/`
2. These extra lines should be indented by 4 spaces relative to the `:param` option, like in the other options there.